internal/middleware/account_test.go

```diff
@@ -0,0 +1,35 @@
+package middleware_test
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+	"github.com/RHEnVision/provisioning-backend/internal/dao/stubs"
+	"github.com/RHEnVision/provisioning-backend/internal/middleware"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccountMiddleware(t *testing.T) {
+	ctx := identity.WithIdentity(t, context.Background())
+	ctx = stubs.WithAccountDaoOne(ctx)
+
+	req, err := http.NewRequestWithContext(ctx, "GET", "/api/provisioning/v1/pubkeys", nil)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Maybe it is better to be consistent in our error handling? 
https://github.com/RHEnVision/provisioning-backend/blob/72c72ff2e09978d6b926ae2f0916a64295fa3054/internal/services/pubkey_service_test.go#L37
Not sure what is the better option